change/@graphitation-ts-codegen-ec02dece-b9be-4f4d-8ba9-362dd9722785.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Add ability to specify model scope",
+  "packageName": "@graphitation/ts-codegen",
+  "email": "mnovikov@microsoft.com",
+  "dependentChangeType": "patch"
```

Comment:
Should be minor probably?

Reply:
Urgh, flipped that one with cli. thanks.